openshift/ansible/__init__.py

```diff
@@ -35,15 +40,25 @@ def __str__(self):
 
 
 class OpenShiftAnsibleModule(AnsibleModule):
+    """
+    Abstraction of an OpenShift Ansible Module.
+    """
+
     def __init__(self, openshift_type):
+        """
```

Comment:
Out of curiosity, do you use any tooling for creating the docstrings? I must admit I'm generally lazy about docstring creation and tend to wait until the interfaces have solidified, since I've been crafting them manually.